pkg/template/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/DataDog/datadog-agent/pkg/template
```

Comment:
Would this module be available for download from outside the repo? If so, should we make it an internal module? That way we do not expose it to outside of this Agent repo 😄

Reply:
I don't think you can have an internal module, internal is only about packages
TBH it's like any module in the agent repo, you can use it at your own risks 😄

Reply:
> TBH it's like any module in the agent repo, you can use it at your own risks 😄

We should put that as a disclaimer somewhere 🤣